src/kudu/consensus/consensus_queue.cc

```diff
@@ -2170,5 +2172,26 @@ string PeerMessageQueue::QueueState::ToString() const {
       active_config ? ", active raft config: " + SecureShortDebugString(*active_config) : "");
 }
 
+std::string PeerMessageQueue::getQuorumId(const RaftPeerPB& peer) {
```

Comment:
nit: confusing to have so many getQuorumId() with different cases.
This case also is inconsistent with rest of file.
I think it is fine to use GetQuorumId or change it to something GetQuorumIdUsingActiveConfig